pkg/api/admin/register.go

```diff
@@ -0,0 +1,36 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+)
+
+const (
+	resourceProviderNamespace = "Microsoft.RedHatOpenShift"
+	resourceType              = "openShiftClusters"
+)
+
+func init() {
+	api.APIs["admin"] = &api.Version{
```

Comment:
Copy from `v20191231preview`. The only difference is this line with the api version name